zap/src/main/java/org/zaproxy/zap/extension/log4j/ZapOutputWriter.java

```diff
@@ -26,6 +26,7 @@
 import org.parosproxy.paros.view.View;
 import org.zaproxy.zap.view.ScanStatus;
 
+@Deprecated
```

Comment:
Should this indicate version or “see”?

Reply:
This is an internal class, I guess we could remove already but just in case deprecated it instead.